session-replacement/src/main/java/com/amadeus/session/repository/redis/NotificationExpirationManagement.java

```diff
@@ -524,6 +524,36 @@ public void close() {
     }
   }
 
+  public void reset() {
+		if (expirationListener != null) {
+
+			try {
+				expirationListener.close(redis);
+				expirationListener = null;
+			} catch (Exception e) {
```

Comment:
Cannot we be more specific on the Exception caught?
Same remark for the other cases where we caught Exception

Small spelling error: generates errors